kie-spring-boot/kie-spring-boot-samples/kie-server-spring-boot-integ-tests-sample/pom.xml

```diff
@@ -24,6 +24,10 @@
       <groupId>org.springframework.boot</groupId>
       <artifactId>spring-boot-starter-security</artifactId>
     </dependency>
+    <dependency>
```

Comment:
Is this dependency needed?

Reply:
yes, this is needed for the DecisionTaskIntegrationTest otherwise it failed with class not found

Reply:
ok